Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,161 @@
+                            .imageScale(.large)
+                    }
+                }
+                VStack {
+                    Text("Elevation: \(viewModel.calibrationElevation?.rounded(.towardZero) ?? viewModel.cameraController.originCamera.location.z?.rounded(.towardZero) ?? 0, format: .number) m")
+                    HStack {
+                        Button {
+                            updateElevation(by: -1)
+                        } label: {
+                            Image(systemName: "minus")
+                                .imageScale(.large)
+                        }
+                        Slider(value: $elevationSliderValue, in: -20...20) { editingChanged in
+                            if !editingChanged {
+                                elevationTimer?.invalidate()
+                                elevationTimer = nil
+                                elevationSliderValue = 0.0
+                            }
+                        }
+                        .onChange(of: elevationSliderValue) { elevation in
+                            guard elevationTimer == nil else { return }
+                            // Create a timer which rotates the camera when fired.
+                            let timer = Timer(timeInterval: 0.1, repeats: true) { _ in
+                                updateElevation(by: joystickElevationDelta)
+                            }
+                            elevationTimer = timer
+                            // Add the timer to the main run loop.
+                            RunLoop.main.add(timer, forMode: .default)
+                        }
+                        Button {
+                            updateElevation(by: 1)
+                        } label: {
+                            Image(systemName: "plus")
+                                .imageScale(.large)
+                        }
+                    }
+                }
+            }
+            .overlay(alignment: .topTrailing) {
+                HStack {
+                    Spacer()
+                    Button {
+                        viewModel.isCalibrating = false
+                        viewModel.setBasemapOpacity(0)
+                    } label: {
+                        Image(systemName: "xmark.circle.fill")
+                            .font(.title3)
+                            .symbolRenderingMode(.hierarchical)
+                            .foregroundStyle(.secondary)
+                            .imageScale(.large)
+                    }
+                    .buttonStyle(.plain)
+                    .frame(alignment: .topTrailing)
+                    .padding()
+                }
+                .padding(.trailing, -20)
+                .padding(.top, -30)
+            }
+            .frame(maxWidth: 350)
+            .padding()
+            .padding(.top, 10)
+            .background(.regularMaterial)
+            .clipShape(RoundedRectangle(cornerRadius: 15))
+            .padding()
```

Comment:
Would it be possible to simplify the layout modifiers here? It would be better not to have multiple paddings unless they are strictly needed.

This might be a good place to use SwiftUI Preview to adjust.

Below is what I came up with. It may still need some adjustment though.

```swift
            // .padding(.top, 5)  // if you want to have some more room for the x button, after the VStacks structure is adjusted
            .overlay(alignment: .topTrailing) {
                Button {
                    viewModel.setBasemapOpacity(0)
                    withAnimation {
                        viewModel.isCalibrating = false
                    }
                } label: {
                    Image(systemName: "xmark.circle.fill")
                        .font(.title3)
                        .symbolRenderingMode(.hierarchical)
                        .foregroundStyle(.secondary)
                        .imageScale(.large)
                }
                .buttonStyle(.plain)
            }
            .frame(maxWidth: 350)
            .padding()
            .background(.regularMaterial)
            .clipShape(RoundedRectangle(cornerRadius: 15))
```

Reply:
I applied your suggestion and added some padding `.padding([.top, .trailing], -5)` for the x button.